pyseir/inference/model_fitter.py

```diff
@@ -418,12 +459,28 @@ def _fit_seir(self, R0, t0, eps, t_break, test_fraction, hosp_fraction, log10_I_
         Returns
         -------
           : float
-            Chi square of fitting model to observed cases and deaths.
+            Chi square of fitting model to observed cases, deaths, and hospitalizations.
         """
         l = locals()
         model_kwargs = {k: l[k] for k in self.model_fit_keys}
-        model = self.run_model(**model_kwargs)
 
+        # Last data point used in Fit
+        last_data_point_used = t0 + t_break + 14 + t_delta_phases + 14
```

Comment:
I don't understand this part.. Why the second 14 here? The ramp is still fit over those 14 days and after.

Reply:
I think I understand now, but this name is very confusing.  I think this has nothing to do with the data right? It is just the end of the *model's* ramp up to the final eps value.  But this also is not the last point used in the fit since eps 2 is fit over all valid data with `t > t0 + t_break + 14 + t_delta_phases`.

Reply:
Yes, sorry I guess a better name would be:
last_data_point_ramp_2, I'll rename in the next PR